internal/pkg/skuba/deployments/ssh/kubeadm.go

```diff
@@ -66,12 +67,16 @@ func kubeadmInit(t *Target, data interface{}) error {
 }
 
 func kubeadmJoin(t *Target, data interface{}) error {
+	api, err := kubernetes.GetAdminClientSet()
```

Comment:
variable use `clientSet` makes consistent.

Reply:
Throughout the code, clientset is used for "k8s.io/client-go/kubernetes" not "github.com/SUSE/skuba/internal/pkg/skuba/kubernetes".